docker-compose-production.yml

```diff
@@ -1,6 +1,11 @@
 version: "3"
 services:
 
+  redis:
```

Comment:
Why do we need Redis container?

Reply:
I think we should use https://docs.celeryproject.org/en/latest/getting-started/brokers/sqs.html
I've added it in this PR: https://github.com/Cloud-CV/EvalAI/pull/1926/